portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,26 +46,32 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     ArrayList<String> comments = new ArrayList<String>();
+    ArrayList<String> emails = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
       String comment = (String) entity.getProperty("comment-string");
+      String email = (String) entity.getProperty("email");
       comments.add(comment);
+      emails.add(email);
     }
 
     List<String> usersComments = new ArrayList<String>();
```

Comment:
(nit) userComments & userEmails. I don't think ppl usually do possession in naming